src/main/java/org/apache/bcel/classfile/Utility.java

```diff
@@ -464,6 +465,21 @@ public static String compactClassName( final String str ) {
     }
 
 
+    /**
+     * Shorten long class names, <em>java/lang/String</em> becomes
+     * <em>java.lang.String</em>,
+     * e.g.. If <em>chopit</em> is <em>true</em> the prefix <em>java.lang</em>
+     * is also removed.
+     *
+     * @param str The long class name
+     * @param chopit flag that determines whether chopping is executed or not
+     * @return Compacted class name
```

Comment:
Hi @markro49 ,
New public methods need `@since 6.4.0` in their Javadocs.

Reply:
Hm, it looks like you just moved it, right? No API changes?

Reply:
Yes; just moved compactClassName(String, boolean) above compactClassName(String, String, boolean).

However, typeSignatureToString replaces the old signatureToString and signatureToString has a new implementation. 
 So typeSignatureToString is a new public method.  should it be marked @since 6.4.0?

Reply:
New public methods should be documented with `@since` Javadoc tags.